tool/RELEASE_INSTRUCTIONS.md

```diff
@@ -235,25 +235,35 @@ of the full release process.
 ### Prepare the release in the `flutter/devtools` repo
 
 1. Find the [DevTools tag](https://github.com/flutter/devtools/tags) that you want
-to perform the cherry-pick release on top of. Then checkout that tag locally. For this
+to perform the cherry-pick release on top of.
+
+For example, if you want current Flutter beta to be updated with fixed DevTools, run in Flutter repo:
+    ```
+    git checkout upstream/beta
+    flutter --version
```

Comment:
this requires that the Flutter git repository is the `flutter` on `PATH`. I think that is the case by default for most of us, but it is worth adding a note here.